views/agency-requests/view_participant.php

```diff
@@ -47,41 +165,153 @@
-                <p class="participant_other"><b>Other: </b> Other items to note</p>
+				<div class="participant_detailed row pb-2">
+				<div class="col-sm-4">
+                <p class="participant_name"><b>Name: </b> <?= $agencyResult['firstname']." ".$agencyResult['middleinit']." ".$agencyResult['lastname'] ?></p>
+				</div>
+				<div class="col-sm-8">
+                <p class="participant_status"><b>Status: </b> <span class="<?=$statuses['class']?>"><?=$statuses['status']?></p>
+				</div>
+				</div>
+				<div class="participant_detailed row pb-2">
+					<div class="col-sm-4">
+						<b>Sex: </b><div><?=$sex=($agencyResult['sex']=="" ? "<i>No records found</i>":$agencyResult['sex'])?></div>
+					</div>
+					<div class="col-sm-4">
+						<b>Race: </b><div><?=$race=($agencyResult['race']=="" ? "<i>No records found</i>":$agencyResult['race'])?></div>
+					</div>
+					<div class="col-sm-4">
+						<b>DOB: </b><div><?=$dob =($agencyResult['dateofbirth']=="" ? "<i>No records found</i>":$agencyResult['dateofbirth'])?></div>
+					</div>
+				</div>
+				
+				<div class="participant_detailed2 row pt-2">
+				<?php
+				while ($rowA = pg_fetch_assoc($agencyInfo)) {
+				?>
+					<div class="col-sm-4">
+					<b>Agency Name: </b>
+						<div> 
+							<?=$agency =($rowA['agency']=="" ? "<i>No records found</i>" : $rowA['agency'])?>
+						</div>
+					</div>
+					
+					<div class="col-sm-4">
+					<b>Agency Contact: </b> 
+					<div> 
+						Phone: 
+						<?=$phone = ($rowA['phone']=="" ? "<i>No records found</i>" :$rowA['phone'])?>
+						Email: 
+						<?=$email = ($rowA['email']=="" ? "<i>No records found</i>" :$rowA['email'])?>
+					</div>
+					</div>
+					
+					<div class="col-sm-4">
+					<b>Consent to Agency: </b> 
+						<div>
+							<?=$consent = ($rowA['hasagencyconsentform'] =="f" || $rowA['hasagencyconsentform'] =="" ? "No" :" Yes")?>
+						</div>
+					</div>
+				<?php
+				}
+				?>
+				</div>
+				<div class="row">
+                <div class="participant_notes col-sm-12">
+					<div> <strong>Notes:</strong> </div>
+					"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
```

Comment:
Remove this lorem ipsum

Reply:
Since this one is going to be fixed in the participant-notes branch, I figured it might be best to just leave it for now.

Reply:
Since this is going to be replaced by the participant notes branch, would you still like me to remove it?

Reply:
I would remove it since we are not sure how long it will be replaced by the participant notes branch. It could say "None" for now or something along those lines.